e2e/utils/waits-utils.ts

```diff
@@ -272,7 +272,7 @@ export async function waitWhileLoading(page: Page, timeout: number = (2 * 60 * 1
   const {waitForRuntime = false}  = opts;
 
   const notBlankPageSelector = '[data-test-id="sign-in-container"], title:not(empty), div.spinner, svg[viewBox]';
-  const spinElementsSelector = `[style*="running spin"], .spinner:empty, [style*="running rotation"]${waitForRuntime ? ':not([aria-hidden="true"])' : ''}`;
+  const spinElementsSelector = `[style*="running spin"], .spinner:empty, [style*="running rotation"]${waitForRuntime ? '' : ':not([aria-hidden="true"])'}`;
```

Comment:
The runtime spinner should be excluded from the check when `waitForRuntime` is false (the default). 
[CI failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/8280/workflows/b8c01109-4830-4a26-995b-0fce02b5fa9d/jobs/125561):
```
TimeoutError: waiting for function failed: timeout 120000ms exceeded
    at new WaitTask (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:505:34)
    at DOMWorld.waitForFunction (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:460:26)
    at Frame.waitForFunction (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:910:32)
    at Page.waitForFunction (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:1291:33)
    at Object.waitWhileLoading (/home/circleci/workbench/e2e/utils/waits-utils.ts:284:14)
```